HelpSource/Classes/Ringz.schelp

```diff
@@ -71,3 +71,28 @@ code::
 )
 ::
 
+Section:: Interaction with sample rate
+
+Ringz (and UGens that are based on it: link::Classes/Klank::, link::Classes/DynKlank:: and link::Classes/Formlet::) are "sample-rate independent" with respect to emphasis::impulses:: at the input. That is, given single-sample impulses, the output signal at different sample rates should be the same frequency and amplitude.
+
+This design has a side effect: If the input is not made of impulses, the output amplitude depends on the sample rate. Higher sample rates produce louder signals, by an amount proportional to the ratio between sample rates.
```

Comment:
"This design has a side effect: If the input is not made of impulses, the output amplitude is directly proportional to the sample rate." might be clearer